<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuditClient</class>
 <widget class="QWidget" name="AuditClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AuditClient</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>641</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string>Connexion au serveur</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>614</width>
      <height>217</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="1">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Num√©ro de port du serveur :</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Adresse IP du serveur :</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLineEdit" name="lineEditAdresseIP">
           <property name="text">
            <string>127.0.0.1</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="2">
          <widget class="QLineEdit" name="lineEditPort">
           <property name="text">
            <string>1025</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButtonQuitter">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>Quitter</string>
           </property>
           <property name="iconSize">
            <size>
             <width>33</width>
             <height>40</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonConnexion">
           <property name="text">
            <string>Connexion</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Etat de la connexion</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEditEtat"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxInformationsClient">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>641</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Informations Client</string>
   </property>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>30</y>
      <width>601</width>
      <height>114</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="pushButtonOrdinateur">
         <property name="text">
          <string>Nom de l'ordinateur distant</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonUtilisateur">
         <property name="text">
          <string>Nom de l'utilisateur</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonArchitecture">
         <property name="text">
          <string>Architecture processeur</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonOs">
         <property name="text">
          <string>OS de l'ordinateur</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLineEdit" name="lineEditOrdinateur"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditUtilisateur"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditArchitecture"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditOs"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
